lib/geordi/capistrano_config.rb

```diff
@@ -11,11 +11,11 @@ def initialize(stage)
 
     def user(server)
       cap2user = deploy_info[/^\s*set\s*:user,\s*['"](.*?)['"]/, 1]
-      cap2user || deploy_info[/^\s*server\s*['"]#{server}['"],.*user.{1,4}['"](.*?)['"]/, 1]
+      cap2user || deploy_info[/^\s*server\s*\(?\s*['"]#{server}['"],.*user.{1,4}['"](.*?)['"]/, 1]
```

Comment:
Should match fine as
```suggestion
      cap2user || deploy_info[/^\s*server\(?\s*['"]#{server}['"],.*user.{1,4}['"](.*?)['"]/, 1]
```
Same below.